### ISSUE TITLE
help with MGB-CPU-01 instructions

### ISSUE THREAD

**Issue by aparrish** [giltesa/Game-Boy-Pocket-USB-C-charging-kit#1]
Hi! I'm trying to install the mod on a MGB-CPU-01 board. The [instructions in the Documentation directory](https://github.com/giltesa/Game-Boy-Pocket-USB-C-charging-kit/blob/master/3.%20Documentation/Incompatibility%20MGB-CPU-01.pdf) are a bit unclear to me. Could you maybe rephrase or rewrite the instructions? As currently written, it seems to me as though steps 1 and 2 are contradictory. Step 1 says to cover both pads with kapton tape, but then step 2 says that you should solder one of the pads... which would be covered with kapton tape at that point, if you followed step 1.

It seems like what I should be doing is soldering a bodge wire from the upper pad of D2 to pin 1 of the DC jack connector, and then covering the lower pad of D2 with kapton tape, along with the bodge wire, but leaving the upper pad of D2 exposed. Is that the case, or is it something else?

**Comment by giltesa** [comment]
Hello Allison,

Sorry, yes, it was a bit confusing. I updated the pdf, I hope it is easier
to understand now:
https://github.com/giltesa/Game-Boy-Pocket-USB-C-charging-kit/blob/master/3.%20Documentation/Incompatibility%20MGB-CPU-01.pdf

[image: imagen.png]




Kind regards,
Alberto

**Comment by aparrish** [comment]
Ah okay, I get it! So the goal is to connect the 5V pad on the USB-C board to pin 1 of the DC jack, and to make sure that nothing gets accidentally soldered to either of the D2 pads. Thanks!
